server/channels/db/migrations/migrations.list

```diff
@@ -495,3 +497,5 @@ channels/db/migrations/postgres/000124_remove_manage_team_permission.down.sql
 channels/db/migrations/postgres/000124_remove_manage_team_permission.up.sql
 channels/db/migrations/postgres/000125_remoteclusters_add_default_team_id.down.sql
 channels/db/migrations/postgres/000125_remoteclusters_add_default_team_id.up.sql
+channels/db/migrations/postgres/000126_scheduled_posts_table.down.sql
+channels/db/migrations/postgres/000126_scheduled_posts_table.up.sql
```

Comment:
nit: can we rename the migration name to `create_scheduled_posts.sql` to match convention? Thanks.

Reply:
Done.